.packit.yaml

```diff
@@ -0,0 +1,21 @@
+specfile_path: beakerlib.spec
+#synced_files:
+#    - beakerlib.spec
+
+upstream_project_name: beakerlib
+downstream_package_name: beakerlib
+
+
+#actions:
+#  create-archive:
+#  - make tarball
+#  get-current-version:
+#  - make version
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
+    - epel-8
```

Comment:
if you want to have different spec files for jobs, you can define a copr_build job for every target (epel7, epel8, fedora-all) and set `specfile_path` in each:
```yaml
- job: copr_build
  metadata:
    targets: epel-8
  specfile_path: epel8/beakerlib.spec
- job: copr_build
  metadata:
    targets: epel-7
  specfile_path: epel7/beakerlib.spec
- job: copr_build
  metadata:
    targets: fedora
  specfile_path: fedora/beakerlib.spec
```